### PR TITLE
Fix to_sandbox.txt axis registry path.

### PR DESCRIPTION
**Pull request by garretrieger**

### CODE REVIEW

File: to_sandbox.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ ofl/notosansthailooped # https://github.com/google/fonts/pull/5111
 lang/languages/vi_Latn.textproto # https://github.com/google/fonts/pull/5105
 
 # API Stuff
-axisregistry/Lib/axisregistry/data/element_grid.textproto # https://github.com/google/fonts/pull/5094
+axisregistry/element_grid.textproto # https://github.com/google/fonts/pull/5094
```
Comment (m4rc1e) [inline to_sandbox.txt:18]: @garretrieger would you like us to apply this path transform for axis registry files going forward? if so, I'll update the ci and our tools to build these lists. We do a similar transform for the lang dir.

Comment (garretrieger) [reply to_sandbox.txt:18]: Yes please, this path is what our tools expect.

**Comment by garretrieger** [comment]
superseded by #5156

File: to_sandbox.txt (hunk 1 above)
Comment (chrissimpkins) [reply to_sandbox.txt:18]: It looks like the CI tests need to be updated in this case.  They fail the path used in #5156

https://github.com/google/fonts/runs/8103625159?check_suite_focus=true

Comment (chrissimpkins) [reply to_sandbox.txt:18]: https://github.com/googlefonts/gftools/issues/603

**Pull request closed without merging**